include/core/jbackend-operation.h

```diff
@@ -108,10 +108,29 @@ gboolean j_backend_operation_unwrap_db_update (JBackend*, gpointer, JBackendOper
 gboolean j_backend_operation_unwrap_db_delete (JBackend*, gpointer, JBackendOperation*);
 gboolean j_backend_operation_unwrap_db_query (JBackend*, gpointer, JBackendOperation*);
 
+gboolean j_backend_operation_unwrap_reset(JBackend* backend,gpointer,JBackendOperation*);
```

Comment:
Shouldn't this be `j_backend_operation_unwrap_db_reset`?

Reply:
the intention was to have one reset function for all backend types